server/utils/CustomError/customError.ts

```diff
@@ -0,0 +1,11 @@
+class CustomError extends Error {
+  public code: number;
+
+  constructor(message: string, code: number) {
+    super(message);
+    this.code = code;
+    this.message = message;
+  }
+}
+
+export default CustomError;
```

Comment:
As you are inside the CustomError folder, then name this file just index.ts, add an index file in the utiles, and import the Custom error from it, not from this file.